modules/renter/contractor/formcontract.go

```diff
@@ -67,7 +67,16 @@ func maxSectors(a modules.Allowance, hdb hostDB, tp transactionPool) (uint64, er
 
 // managedNewContract negotiates an initial file contract with the specified
 // host, saves it, and returns it.
-func (c *Contractor) managedNewContract(host modules.HostDBEntry, numSectors uint64, endHeight types.BlockHeight) (modules.RenterContract, error) {
+func (c *Contractor) managedNewContract(host modules.HostDBEntry, numSectors uint64, endHeight types.BlockHeight) (_ modules.RenterContract, err error) {
+	// Increase Successful/Failed interactions accordingly
+	defer func() {
+		if err != nil {
+			c.hdb.IncrementFailedInteractions(host.PublicKey)
+		} else {
+			c.hdb.IncrementSuccessfulInteractions(host.PublicKey)
+		}
+	}()
```

Comment:
This is still happening too early. It's not a failed interaction technically if the host is too expensive (a separate part of the logic handles that case), and it's definitely not a failed interaction if the wallet returns an error for `c.wallet.NextAddress()`